tests/client.test.ts

```diff
@@ -1,3 +1,4 @@
+import { afterAll, expect, test, describe, beforeEach } from 'vitest';
```

Comment:
Vitest by default and recommended way is to have explicit imports for these methods.